### PR TITLE
Fix undefined variables messages

### LINKED ISSUE #3988: Undefined variable $datequery in function network_tabs
Notice: Undefined variable: datequery in /home/kirgroup/htdocs/www/mod/network.php on line 895

### LINKED ISSUE #3989: Undefined variable $cid in function network_init
Notice: Undefined variable: cid in /home/kirgroup/htdocs/www/mod/network.php on line 159

### LINKED ISSUE #3990: Undefined variable $ret in Config::set()
Notice: Undefined variable: ret in /home/kirgroup/htdocs/www/src/Core/Config.php on line 164

Notice: Undefined variable: ret in /home/kirgroup/htdocs/www/src/Core/Config.php on line 168

### PR DESCRIPTION
**Pull request by MrPetovan**
Fixes #3988.
Fixes #3989.
Fixes #3990

### CODE REVIEW

File: mod/network.php [hunk 1]
```diff
@@ -883,17 +884,19 @@ function networkThreadedView(App $a, $update = 0) {
...
 	list($no_active, $all_active, $postord_active, $conv_active, $new_active, $starred_active, $bookmarked_active, $spam_active) = network_query_get_sel_tab($a);
+
 	// if no tabs are selected, defaults to comments
-	if ($no_active=='active') $all_active='active';
+	if ($no_active == 'active') {
+		$all_active = 'active';
+	}
 
-	$cmd = (($datequery) ? '' : $a->cmd);
-	$len_naked_cmd = strlen(str_replace('/new','',$cmd));
```
Comment (annando) [inline mod/network.php]: Is "datequery" and "len_naked_cmd" never used?

Comment (MrPetovan) [reply mod/network.php]: Nope...

Comment (annando) [reply mod/network.php]: "Nope" means "yes"? (Think of double negation) :smile:

Comment (MrPetovan) [reply mod/network.php]: It does, I think? Those variables aren’t used ever.

Comment (MrPetovan) [reply mod/network.php]: It does, I think? Those variables aren’t used ever.

Comment (fabrixxm) [reply mod/network.php]: `$datequery` (and also `$cid`) are defined in [networkThreadedView()](https://github.com/MrPetovan/friendica/blob/58ac3532b74017802e4027400a3fe6b4eb76efe2/mod/network.php#L518-L540)

Could be that the code was splitted?

Comment (MrPetovan) [reply mod/network.php]: Maybe, but it still doesn't make sense in `network_tabs()`.

**Pull request merged**